cmd/guacone/cmd/oci.go

```diff
@@ -88,7 +90,7 @@ var ociCmd = &cobra.Command{
 		// Set emit function to go through the entire pipeline
 		emit := func(d *processor.Document) error {
 			totalNum += 1
-			err := ingestor.Ingest(ctx, d, opts.graphqlEndpoint, transport, csubClient, opts.queryVulnOnIngestion)
+			err := ingestor.Ingest(ctx, d, opts.graphqlEndpoint, transport, csubClient, opts.queryVulnOnIngestion, opts.queryLicenseOnIngestion)
```

Comment:
I'm seeing this pattern repeated throughout of adding booleans for some behavior changes. Would this be a good time to change the function signature to take opts as a single parameter instead of each bool separately?

Reply:
yes great point. We can clean this up in a future pr